src/sentry/static/sentry/app/components/onboardingWizard/task.tsx

```diff
@@ -133,7 +134,11 @@ function Task({router, task, onSkip, onMarkComplete, forwardedRef, organization}
         {IncompleteMarker}
         {task.title}
       </Title>
-      <Description>{`${task.description}. ${task.detailedDescription}`}</Description>
+      <Description>{`${task.description}. ${task.detailedDescription ??
+        ''}`}</Description>
+      {isSecondPlatformPending(task) && (
```

Comment:
Ah this is unfortunate that we have to specialize it :( Is it not possible to add this to the `task.description`?